modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -221,7 +223,75 @@ public ReturnType call(final String[] args, Cfg cfg) {
                 // no return, allow for other parameters combined with -p
             }
 
-            // 4. can be influenced by the -n, -d, -p arguments above
+            // determine the sync compact configuration, can be combined with the -n, -d, -c, -p, -i
+            // arguments
+
+            if (options.getCompact() != null) {
+
+                CfgSync cfgSync = cfg.getSync();
+                String[] parameters = options.getCompact();
+
+                if (parameters.length == 1) {
+
+                    if (!parameters[0].equalsIgnoreCase("true")
+                            && !parameters[0].equalsIgnoreCase("false")) {
+                        System.out.println("enabled value must be boolean");
```

Comment:
I think instead of "value must be boolean" it would be better to print "value must be true or false"